contracts/system-contracts/hedera-token-service/examples/hrc-904/TokenReject.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity >=0.5.0 <0.9.0;
+
+import {HederaResponseCodes} from "../../../HederaResponseCodes.sol";
+import {HederaTokenService} from "../../HederaTokenService.sol";
+import {IHederaTokenService} from "../../IHederaTokenService.sol";
+pragma experimental ABIEncoderV2;
+
+contract TokenReject is HederaTokenService {
```

Comment:
Curious, why the separation from HRC904Contract.sol?

Reply:
Some of the subtasks on the epic state "Add tests that verify IHRC904 rejectTokens targeted to an EOA are functional when called by a contract". This is the example contract I planned to use to cover the test cases and it also allows me to do a batch reject.